cl/citations/tests.py

```diff
@@ -17,6 +19,12 @@
 )
 from factory import RelatedFactory
 from lxml import etree
+from rest_framework.status import (
+    HTTP_200_OK,
+    HTTP_300_MULTIPLE_CHOICES,
+    HTTP_400_BAD_REQUEST,
+    HTTP_404_NOT_FOUND,
+)
```

Comment:
I actually prefer pulling these from the `http` library, since they're there. If you want to include that refactor across the entire project in this PR, I won't stop you.